mofacts/client/views/home/profile.js

```diff
@@ -294,7 +294,12 @@ Session.set('speechAPIKey', null);
 Template.profile.rendered = async function() {
   sessionCleanUp();
   Session.set('showSpeechAPISetup', true);
-  allTdfs = await meteorCallAsync('getAllTdfs');
+  if(Meteor.user().profile.loginMode === 'southwest') {
+    const curSectionId = Meteor.user().profile.curClass.sectionId;
+    const allTdfs = await meteorCallAsync('getTdfsAssignedToStudent', Meteor.userId(), curSectionId);
```

Comment:
This variable would be out of scope for the rest of the function throwing errors and breaking profile

Reply:
true